packages/terra-navigation-layout/package.json

```diff
@@ -22,17 +22,14 @@
   },
   "homepage": "https://github.com/cerner/terra-framework#readme",
   "devDependencies": {
-    "react-redux": "^5.0.4",
```

Comment:
Why the removal of all the redux packages?

Reply:
?

Reply:
I just double checked. If you go through the examples and source code,  react-redux or redux is not being used anywhere.

Reply:
@tbiethman @dkasper-was-taken could you verify that I'm not missing something?